pyproject.toml

```diff
@@ -48,40 +63,37 @@ classifiers = [
 python = "^3.8"
 screenpy = ">=4.0.2"
 requests = ">=2.26.0"
-types-requests = "2.25.0"
+types-requests = ">=2.26.0"
 
 
 # convenience packages for development of screenpy_requests only
 black = {version = "*", optional = true}
 coverage = {version = "*", optional = true}
-flake8 = {version = "*", optional = true}
 isort = {version = "*", optional = true}
```

Comment:
Do we want to remove `isort` now that we figured `ruff`'s version out?

Reply:
Good call.  Removed.